usr/share/rear/build/default/990_verify_rootfs.sh

```diff
@@ -80,6 +80,13 @@ if test "$BACKUP" = "TSM" ; then
     # see https://github.com/rear/rear/issues/1533
     export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$TSM_LD_LIBRARY_PATH
 fi
+#TODO remove
+#if test "$BACKUP" = "FDRUPSTREAM" ; then
+    # Use a FDRUPSTREAM-specific LD_LIBRARY_PATH to find FDR libraries
```

Comment:
This change here does not belong to this pull request
and must be removed from this bull request.

It belongs actually to https://github.com/rear/rear/pull/2296

Reply:
@jsmeix 
9f21407b
oops,
i removed this commit